.bmad/bmb/workflows/create-agent/agent-command-patterns.md

```
@@ -0,0 +1,759 @@
+
+```xml
+<item cmd="*brainstorm">Brainstorming Session</item>
+<item cmd="*ideate">Ideation Workshop</item>
+<item cmd="*storytell">Story Creation</item>
+```
+
+## Command Menu Presentation
+
+### How Commands Display
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)